code/common/heartbeat.q

```diff
@@ -7,13 +7,11 @@
 // Override the processwarning and processerror functions to implement required behaviour when warnings or errors are encountered
 // Override warningperiod and errorperiod functions to have bespoke warning and error periods for different process types
 // Use storeheartbeat function in the upd function to process heartbeats
-
 \d .hb
-
```

Comment:
Bring the newlines back either side of `\d .hb`, will help keep the namespace change more visible.